src/shared/tpm2-util.c

```diff
@@ -1264,6 +1264,7 @@ int tpm2_make_luks2_json(
                 size_t blob_size,
                 const void *policy_hash,
                 size_t policy_hash_size,
+                int flags,
```

Comment:
having `signed` flags is a bit weird, due to sign extension and stuff. We usually do flags fields as enum these days, or at least as "unsigned" or "uint32_t" or so.